### ISSUE TITLE
Auto phosphor

### ISSUE THREAD

**Issue by thrust26** [stella-emu/stella#1009]
The idea is to automatically enable phosphor when flicker is detected by Stella. And maybe disable it too, when there is none.

For that could compare the positions of sprites, missiles and the ball over multiple frames. E.g. if a sprite alternates positions (with some margin due to movement), this would indicate flicker. Note: This has to consider reused sprites too!

**Comment by sa666666** [comment]
This has been suggested before, but we never did come up with a reliable way to detect it.  If you can do so, then great :smiley:

**Comment by thrust26** [comment]
> This has been suggested before...

It was? Where?

**Comment by sa666666** [comment]
I think before you guys came onboard, and before we started documenting our suggestions.

**Comment by thrust26** [comment]
Anymore you do remember?

**Comment by sa666666** [comment]
No.  That was 6 or 7 years ago, there's no chance I can remember back that far.  Unless it's documented, I can't remember things from 6 months ago, let alone years.

**Comment by sa666666** [comment]
Anyway, I remember the discussion didn't get far, so there's not much extra info to tell.

**Comment by thrust26** [comment]
I checked https://stella-emu.github.io/todo.html hoping for some findings.

I think the KidVid can be removed.

**Comment by sa666666** [comment]
KidVid is not fully supported yet, but enough so that it can be removed from the TODO list (the remaining missing KidVid stuff already has an issue created).
